### ISSUE TITLE
bug: Logo missing alt text

### ISSUE THREAD

**Issue by cathy-codes** [WomenCodingCommunity/WomenCodingCommunity.github.io#110]
**Task**
Add alt text for the logo located in the header and footer. This is important so that screen reader users can understand the image, otherwise it can be picked up as a "graphic" or have the name of the file read out.

**Comment by cathy-codes** [comment]
Proposed alt text:
- Header: "Women Coding Community home" since it's a link that links to the home page
- Footer: "Women Coding Community logo"

**Comment by MitaliShah** [comment]
Hello, I would like to contribute if the issue is still available. Thank you!

**Comment by cathy-codes** [comment]
Hey @MitaliShah, yes it's still available. Feel free to work on it!

**Comment by MitaliShah** [comment]
@cathy-codes Thanks!
